test/action_controller/json/include_test.rb

```diff
@@ -74,37 +78,58 @@ def test_render_without_include
           assert_equal(expected, response)
         end
 
+        def test_render_with_default_includes_hash
+          with_default_includes INCLUDE_HASH do
```

Comment:
this is a bit misleading, right, because you'd never have a default includes of `posts.comments`, right? the more common, scenarios, cc @beauby would be `'*'`, `''`, `'*.*'`, I think?